features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/ImagePrivacy.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay
+
+/**
+ * Defines the Session Replay privacy policy when recording images.
+ * @see ImagePrivacy.ALL
+ * @see ImagePrivacy.CONTEXTUAL
+ * @see ImagePrivacy.NONE
+ */
+enum class ImagePrivacy {
+    /**
+     * All images will be recorded, including those downloaded from the Internet during app runtime.
+     */
+    ALL,
+
+    /**
+     * Mask images that we consider to be contextual.
```

Comment:
this is a bit weird for me, if I do the corelation with the first option `ALL` for me `CONTEXTUAL` would mean the opposite (we record only contextual images)

Reply:
I've changed these values to the ones that we decided upon in the RFC (MASK_NONE, MASK_ALL, MASK_CONTENT)